src/SwiftPackage.ts

```diff
@@ -78,7 +78,7 @@ export class PackageResolved {
                         pin.state
                     )
             );
-        } else if (this.version === 2) {
+        } else if (this.version === 2 || this.version === 3) {
```

Comment:
I'm assuming the actual JSON layout hasn't changed, just the version number?

Reply:
JSON layout of the bits we care about hasn't changed